docs/docsite/rst/user_guide/playbooks_vars_facts.rst

```diff
@@ -636,7 +636,7 @@ Information about Ansible: magic variables
 
 You can access information about Ansible operations, including the python version being used, the hosts and groups in inventory, and the directories for playbooks and roles, using "magic" variables. Like connection variables, magic variables are :ref:`special_variables`. Magic variable names are reserved - do not set variables with these names. The variable ``environment`` is also reserved.
 
-The most commonly used magic variables are ``hostvars``, ``groups``, ``group_names``, and ``inventory_hostname``. With ``hostvars``, you can access variables defined for any host in the play, at any point in a playbook. You can access Ansible facts using the ``hostvars`` variable too, but only after you have gathered (or cached) facts.
+The most commonly used magic variables are ``hostvars``, ``groups``, ``group_names``, and ``inventory_hostname``. With ``hostvars``, you can access variables defined for any host in the play, at any point in a playbook. You can access Ansible facts using the ``hostvars`` variable too, but only after you have gathered (or cached) facts. Note that global variables, for example variables defined in the playbook ``vars`` section, are not mapped to ``hostvars``.
```

Comment:
There are no 'global variables', variables defined in play are that, play variables and not defined for the specific host.

Note that play variables do not survive across plays in the same run, while 'host scoped variables' do.

Reply:
Thank you, I rephrased